### PR TITLE
Add direct share links for copy+paste

### PR DESCRIPTION
**Pull request by dbalatero**

### CODE REVIEW

**Comment by dbalatero** [comment]
screenshot 2016-04-11 12 32 23

File: app/views/success/show.html.rb [hunk 1]
```diff
@@ -42,6 +42,16 @@ def content
     end
 
     full_row do
+      p do
+        text "Direct share link for #{States.with_code(state).name} state: "
```
Comment (bion) [inline app/views/success/show.html.rb]: let's change the exposed `state` to be a `State` struct in the controller

Comment (dbalatero) [inline app/views/success/show.html.rb]: okay

**Comment by bion** [comment]
:ship:

**Pull request merged**